net/test/testdata/aci_cfg.yml

```diff
@@ -0,0 +1,17 @@
+# This is the configuration for baremetal used for testing aci installer
+# XXX: for now we have one setup so hardcoding the details, we need to
+# revisit this when we add more aci test setups
+
+CONNECTION_INFO:
+      contiv-aci-server4:
+        control: eth3
+        data: eth7
+      contiv-aci-server5:
+        control: eth3
+        data: eth7
+APIC_URL: "https://contiv-aci-ifc1:443"
+APIC_USERNAME: "admin"
+APIC_PASSWORD: "ins3965!"
```

Comment:
yeah, I think they can be take from environment but I don't have a way to generate this file...note that the credentials are specific to the private setup in our lab network.

I intentionally let this entire file be a static yml for now given a single test setup. May be for now we can replace them with some more dumb/fake values as the installer doesn't really cares?